src/convertToObject.js

```diff
@@ -3,16 +3,32 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example in
+ * [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties (see an exampl
+ * in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+  const getData = sourceString.trim().split(';');
+
+  getData.forEach((item) => {
+    if (item !== '') {
+      const [key, value] = (item.trim().split(':'));
+
+      if (key !== undefined && value !== undefined) {
```

Comment:
Consider how you can rewrite this condition so that you don't explicitly write a check for undefinеd, as this is bad practice
```suggestion
      if (key !== undefined && value !== undefined) {
```

Reply:
smthng like this :
 if (!key &&& !value) ?

Reply:
yeah, exactly